com.cubrid.cubridmanager.core.testfragment/.classpath

```diff
@@ -4,5 +4,6 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="resource"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.cubrid.common.ui"/>
```

Comment:
Yes. We are now using CMHostNodePersistManager.getInstance().addServer in TestRawScript.java. This class needs to know about CubridServer (used by CMHost....addServer), so it needs to include com.cubrid.common.ui (where CubridServer is).